src/chunks/admin-template-chunk/application-template-chunk/ApplicationTemplateChunk.tsx

```diff
@@ -0,0 +1,616 @@
+"use client";
+import Image from "next/image";
+import React from "react";
+import date from "public/images/date_icon.svg";
+import useApplicationTemplateChunk from "./useApplicationTemplateChunk";
+import searchicon from "public/images/searchicon.svg";
+import filtericon from "public/images/filter_icon.svg";
+import { alldata } from "@/utils/data/dashboardapplication";
+import FindjobPagination from "@/components/pagination/FindjobPagination";
+
+const ApplicationTemplateChunk = () => {
+  const {
+    ITEMS_PER_PAGE,
+    paginate,
+    currentPage,
+    currentdata,
+    currentassesmentdata,
+    currentinterviewdata,
+    currentoffersdata,
+    currentunsuitabledata,
+    currentreviewdata,
+    filter,
+    setFilter,
+  } = useApplicationTemplateChunk();
+
+  return (
+    <section className="py-5 grid grid-flow-row gap-10 max-w-full h-full ">
+      <div className="flex justify-between">
+        <div className="flex flex-col gap-[2px]">
+          <h2 className="text-[24px] font-[600] leading-[40px] text-[#1B1B1B] font-clash ">
+            Don’t give up, Tosin
+          </h2>
+          <p className="text-[#515B6F] opacity-60 font-[400] font-epilogue leading-[30px] text-[16px]">
+            Here are your applications so far
+          </p>
+        </div>
+
+        <div className="border-[1px] border-[#1B1B1B66] gap-[10px] p-[10px] w-[112px] h-[40px] flex items-center justify-center">
+          <h1 className="font-[600] text-[16px] leading-[16px] font-epilogue text-[#1B1B1B]">
+            July
```

Comment:
work on this implementation such that when clicked, it shows a calendar where one can select date. make it functional